.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,110 @@
+            core.setOutput('contributor-username', pr.user.login);
+            
+      - name: Send bounty data to BLT
+        if: steps.check-issues.outputs.has-bounty == 'true'
+        env:
+          ISSUE_NUMBER: ${{ steps.check-issues.outputs.issue-number }}
+          BOUNTY_AMOUNT: ${{ steps.check-issues.outputs.bounty-amount }}
+          CONTRIBUTOR_USERNAME: ${{ steps.check-issues.outputs.contributor-username }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          BLT_API_TOKEN: ${{ secrets.BLT_API_TOKEN }}
+          API_BASE_URL: ${{ secrets.API_BASE_URL || 'https://owasp.org' }}
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)